doc/contributor/project-layout.md

```diff
@@ -0,0 +1,33 @@
+# Project Layout
+
+Each folder in the TruffleRuby project is described below:
+
+## `/bench/`
+Files for running benchmarks. See the `jt benchmark` command.
+
+## `/bin/`
```

Comment:
The leading slash looks strange to me, I would prefer `bin/` as it's not a path from the system root directory.

Reply:
Fixed, removed leading slashes.